src/main/java/taxi/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package taxi.service;
+
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.ManufacturerDao;
+import taxi.db.Storage;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return Optional.of(Storage.manufacturers
+                .get(Math.toIntExact(id))).get();
```

Comment:
we can't use Storage at service layer, let's use manufacturerDao instead

Reply:
done